packages/@aws-cdk/aws-codepipeline-actions/lib/s3/deploy-action.ts

```diff
@@ -29,6 +29,23 @@ export interface S3DeployActionProps extends codepipeline.CommonAwsActionProps {
    * The Amazon S3 bucket that is the deploy target.
    */
   readonly bucket: s3.IBucket;
+
+  /**
+   * The specified canned ACL to objects deployed to Amazon S3.
+   * This overwrites any existing ACL that was applied to the object.
+   *
+   * @default - the original object ACL
+   */
+  readonly cannedAcl?: s3.BucketAccessControl;
```

Comment:
This should be called `accessControl` to be consistent with [`BucketProps.accessControl`](https://docs.aws.amazon.com/cdk/api/latest/docs/@aws-cdk_aws-s3.Bucket.html#accesscontrol)

Reply:
Done